### PR TITLE
refactor: remove newMarketPage feature flag

### PR DESCRIPTION
**Pull request by therealemjy**
## Changes

- remove `newMarketPage` feature flag
- remove old Market page component
- remove `useOperationModal` hook
- move `OperationForm` container inside new Market page component

### CODE REVIEW

File: apps/evm/src/pages/Market/LidoPoolMarket/index.tsx [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
 
 import { useGetChainMetadata } from 'hooks/useGetChainMetadata';
-import { useIsFeatureEnabled } from 'hooks/useIsFeatureEnabled';
 
 import MarketLoader from 'containers/MarketLoader';
 import { Market } from '../Market';
-import MarketOld from '../MarketOld';
 
-const StakedEthPoolMarket: React.FC = () => {
+const LidoPoolMarket: React.FC = () => {
```
Comment (gleiser-oliveira) [inline apps/evm/src/pages/Market/LidoPoolMarket/index.tsx]: Shouldn't we keep this as StakedEthPoolMarket? It's actually the file name that is outdated

File: apps/evm/src/pages/Market/Market/OperationForm/RepayForm/index.tsx [hunk 2]
```diff
@@ -552,8 +552,11 @@ const RepayForm: React.FC<RepayFormProps> = ({
         token: fromToken,
       });
 
+      console.log('CALLED');
+
       // Handle repay flow
       if (!isUsingSwap) {
+        console.log('no swap');
```
Comment (gleiser-oliveira) [inline apps/evm/src/pages/Market/Market/OperationForm/RepayForm/index.tsx]: A couple of logs here

Comment (therealemjy) [reply apps/evm/src/pages/Market/Market/OperationForm/RepayForm/index.tsx]: I need to pay more attention to these, you've found logs in all my latest PRs... Thanks for the catch.

File: apps/evm/src/pages/Market/LidoPoolMarket/index.tsx (hunk 1 above)
Comment (therealemjy) [reply apps/evm/src/pages/Market/LidoPoolMarket/index.tsx]: Actually, it seems this file isn't used at all. You're right that the name was changed to `StakedEthPoolMarket`, but we already have a file for it. I'll just remove `LidoPoolMarket`

File: apps/evm/src/pages/Market/Market/OperationForm/RepayForm/index.tsx (hunk 2 above)
Comment (gleiser-oliveira) [reply apps/evm/src/pages/Market/Market/OperationForm/RepayForm/index.tsx]: Just to ping you, they are still there! 😆

Comment (therealemjy) [reply apps/evm/src/pages/Market/Market/OperationForm/RepayForm/index.tsx]: Oh man... Now they're gone 😅

**Review by gleiser-oliveira — APPROVED** [review]

**Pull request merged**